vertical-pod-autoscaler/recommender/recommender.go

```diff
@@ -201,14 +239,8 @@ func (r *recommender) runOnce() {
 	glog.V(3).Infof("Recommender Run")
 	r.loadVPAs()
 	r.loadPods()
-
-	metricsSnapshots, err := r.metricsClient.GetContainersMetrics()
-	if err != nil {
-		glog.Errorf("Cannot get containers metrics. Reason: %+v", err)
-	}
-	for n, snap := range metricsSnapshots {
-		glog.V(3).Infof("ContainerMetricsSnapshot #%v: %+v", n, snap)
-	}
+	r.loadMetrics()
+	glog.V(3).Infof("Current ClusterState:  %+v", r.clusterState)
```

Comment:
ClusterState is a big object - it has data for all the containers in the cluster and for each of them some histograms, peaks list etc. I think this log is polluting the logs and will be extremely hard to read. I'd much rather see some summary (like the one I added in line 72 above: just how many samples we're adding, without the details of what they are exactly).

Reply:
good point!
I added a summary with just a total number of pods and vpa objects in ClusterState.
Plus a summary of metrics being added has been already defined in line [163](https://github.com/kubernetes/autoscaler/pull/578/files#diff-628cd012db9dc0c7d3209b8abffc2ff5R163), just after metrics load process.